### DISCUSSION TITLE
Percentage increase and decrease

### DISCUSSION THREAD

**Discussion by sergeevabc** [Qalculate/libqalculate#613] [Q&A] [answered]
Dear @hanna-kn,
sometimes I manage to reduce something bloated, for example, reduce the image weight without noticeable loss of quality, and then I want to express the success in percentages.

![2024-0125-2126 omnicalculator percentage decrease](https://github.com/Qalculate/libqalculate/assets/373718/e52d6128-4e2a-4d74-b91a-adf7710b2937)

[Percentage decrease calculator](https://www.omnicalculator.com/math/percentage-decrease) says here that the image weight was reduced by 84%.
What do I need to enter after `qalc.exe` to get that value?
I tried `qalc.exe percentage difference between 406958 and 66873`.

**Comment by hanna-kn** [comment, marked as answer]
The GUIs (qalculate-gtk and qalculate-qt) provides a dialog for simple percentage calculation.

With qalc you will have to resort to simple division, e.g. `64949/406958 − 1`. This will return −0.84. Multiply with 100 or use `to %` (or `->%`), e.g. `qalc 64949/406958-1 to %`, to get the result in percent.
